dftimewolf/lib/collectors/grr_hosts.py

```diff
@@ -147,11 +154,11 @@ def _await_flow(self, client, flow_id):
         status = client.Flow(flow_id).Get().data
       except grr_errors.UnknownError:
         msg = 'Unable to stat flow {0:s} for host {1:s}'.format(
-            flow_id, self.host)
+            flow_id, client.data.os_info.fqdn.lower())
```

Comment:
Is client.data.os_info.fqdn always set? I suspect probably not, so we'll get another exception here. Consider refactoring this.

Reply:
There's a lot of logic assuming valid fqdns spread around the code. I think this might be worth revisiting in a future PR.